src/migrations/20220528143630-dont-cascade-environment-deletion-to-apitokens.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+          ALTER TABLE api_tokens DROP CONSTRAINT api_tokens_environment_fkey;
+      `,
+        cb,
+    );
+};
+
+exports.down = function (db, cb) {
+    db.runSql(
+        `
+          ALTER TABLE api_tokens DROP CONSTRAINT api_tokens_environment_fkey;
```

Comment:
Should both these commands be the same? If the up command drops the constraint, shouldn't the down command create that constraint?